R/ggsurvfit.R

```diff
@@ -73,13 +73,17 @@ ggsurvfit <- function(x, type = "survival",
   # prep data to be passed to ggplot() -----------------------------------------
   df <-
     tidy_survfit(x = x, type = type) %>%
+    dplyr::filter(type != "cloglog" | .data$time > 0) %>%
```

Comment:
@avishaitsur can you explain what the purpose of this addition is? Thank you!

Reply:
The transformation in log minus log survival is `cloglog = function(y) log(-log(y))`.
In time 0, by definition, the survival `y` is 1, and therefore, the result is undefined (log of 1 is 0, and log of -0 is undefined).
For this reason, the transformation and the visual do not apply to time 0.

Reply:
Of course, thank you!

Reply:
I wonder the best way to handle this as `survfit()` allows negative times. 🤔

Reply:
The transformation and the visual are implemented in the `survival` and `survminer` packages. Maybe we could explore their solutions?

Reply:
worst case, we get a NaN or Inf value and it won't appear on the plot.

I am going to go ahead and merge this now (assuming the current tests complete without error). I am also going to make a release